src/main/resources/assets/seedmapper/lang/hi_in.json

```diff
@@ -0,0 +1,82 @@
+{
+  "commands.exceptions.dimensionNotSupported": "आयाम \"%s\" समर्थित नही है।",
+  "commands.exceptions.versionNotFound": "संस्करण \"%s\" नही मिला",
+  "commands.exceptions.unknownConfig": "अज्ञात कॉन्फिग \"%s\".",
+  "commands.exceptions.nullPointerException": "\"%s\" null है",
+  "commands.exceptions.invalidDimension": "अमान्य आयाम।",
+  "commands.exceptions.biomeNotFound": "बायोम \"%s\" नही मिली।",
+  "commands.exceptions.structureNotFound": "सरंचना \"%s\" नही मिली",
+  "commands.exceptions.lootNotFound": "आइटम \"%s\" नही मिला",
+  "commands.exceptions.lootItemNotFound": "लूट आइटम \"%s\" नही मिला।",
+  "commands.exceptions.blockNotFound": "ब्लॉक \"%s\" नही मिला।",
+  "commands.exceptions.expectedWithOrWithout": "Expected \"with\" or \"without\".",
+  "commands.exceptions.unknownEnchantment": "अज्ञात एनचेंटमेन्ट \"%s\".",
+  "commands.exceptions.levelOutOfBounds": "स्तर सीमा से बाहर",
+  "commands.exceptions.incompatibleEnchantment": "असंगत एनचेंटमेन्ट।",
+  "commands.exceptions.incompleteArgument": "अपूर्ण आर्गुमेंट",
+
+  "arguments.exceptions.tooFewArguments": "बहुत कम आर्गुमेंट।",
+
+  "command.checkSeed.using": "SeedMapper सीड %d को यहाँ से उपयोग कर रहा है",
+  "command.checkSeed.fromSeeds": "the \"seeds\" config",
+  "command.checkSeed.fromDatabase": "ऑनलाइन डेटाबेस",
+  "command.checkSeed.fromSeed": "the \"seed\" config",
+  "command.checkSeed.fromSource": "the /sm:source command",
+  "command.checkSeed.none": "SeedMapper उपयोग करने के लिए सीड नहीं खोज सका।",
+
+  "command.clearscreen.empty": "कोई रेंडर हटा नहीं सका।",
+  "command.clearscreen.success": "%d रेंडर हटा दिए गए।",
+
+  "command.config.setAutomate": "Set \"automate\" to %s.",
+  "command.config.getAutomate": "\"automate\" is set to %s.",
+  "command.config.toggleAutomate": "Toggled \"automate\" to %s.",
+  "command.config.setSeed": "Set \"seed\" to %d.",
+  "command.config.getSeed": "\"seed\" is set to %d.",
+  "command.config.getSeed.null": "\"seed\" is null.",
```

Comment:
Why aren't these translated?

Reply:
Translating these was kind of hard and not really making any sense, some languages have not translated these too.

Reply:
What precisely is hard about translating these sentences?